### PR TITLE
Convert ListItemTag to live code examples

### PR DESCRIPTION
**Pull request by priyankakmEaton**
Fixes #5799

#### Changes proposed in this Pull Request:

-Convert images to live RN code (using react-native-web)

-Remove unused image assets

-Convert markdown code blocks to PrismJS code blocks

-Use PrismJS to highlight targeted lines in the code example blocks

-Include buttons to copy to clipboard and view full github source

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItemExample.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import * as Colors from '@brightlayer-ui/colors';
+import {ExampleShowcase} from '../../../shared';
+import {View} from 'react-native';
+
+export const ListItemTagUsingInfoListItemExample = (): JSX.Element => (
+  <ExampleShowcase sx={{display: 'flex', justifyContent: 'center'}}>
+    <InfoListItem
+      icon={{name: 'battery-charging-full'}}
+      iconColor="#528AAE"
+      style={{backgroundColor: Colors.white[100]}}
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItemExample.tsx:15]: you can remove style={{backgroundColor: Colors.white[100]}}

File: docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItem.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        />
+      </View>
+    }
+  />
+  `;
+
+export const ListItemTagUsingInfoListItem = (): JSX.Element => (
+  <Box>
+    <ListItemTagUsingInfoListItemExample />
+    <CodeBlock code={codeSnippet} language="jsx" dataLine="9-14" />
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItem.tsx:34]: please update to dataLine="9-13, 15-19"

File: docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItem.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+import {CodeBlock, CodeBlockActionButtonRow} from '../../../shared';
+import {ListItemTagUsingInfoListItemExample} from './ListItemTagUsingInfoListItemExample';
+
+const codeSnippet = `
+  <InfoListItem
+    icon={{name: 'battery-charging-full'}}
+    iconColor="#528AAE"
+    style={{backgroundColor: Colors.white[100]}}
```
Comment (JeffGreiner-eaton) [inline docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItem.tsx:10]: please remove style={{backgroundColor: Colors.white[100]}} from snippet and format the snippet

File: docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItem.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+const codeSnippet = `<InfoListItem
+    icon={{name: 'battery-charging-full'}}
+    iconColor="#528AAE"
+    title="Info List Item"
+    subtitle="with a ListItemTag component to the right"
+    rightComponent={
+      <View style={{flexDirection: 'row'}}>
+          <ListItemTag
+              label="Build Passing"
+              backgroundColor={Colors.green[300]}
```
Comment (ektaghag-eaton) [inline docs/src/componentDocs/ListItemTag/examples/ListItemTagUsingInfoListItem.tsx:15]: Can we use Hex code instead colors.green[300]? RN components use MD3 colors and the react-native docs app is in react which uses MD2 colors.

**Review by ektaghag-eaton — APPROVED** [review]

**Pull request merged**